packages/lambda-tiler/src/routes/tile.ts

```diff
@@ -132,16 +132,16 @@ export const TileRoute = {
     const sources: Sources = {};
     for (const [key, value] of Object.entries(style.sources)) {
       if (value.type === 'vector') {
-        if (value.url.includes(host)) {
-          const url = new URL(value.url);
```

Comment:
what happens with `https://basemaps.linz.govt.nz/` does this duplicate `/`? I normally use `fsa.join` to join urls

`path.join` with urls work very well either